profile-management/OktaProfileManagement/AppDelegate.swift

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import OktaOidc
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    lazy var oktaOidc: OktaOidc = {
+        if let configForUITests = AppDelegate.shared.configForUITests {
+            return try! OktaOidc(configuration: OktaOidcConfig(with: configForUITests))
```

Comment:
Try not to force-unpack via `try`, as if the `OktaOidc` object throws, it'll cause the application to crash.

Suggestion - surround this in a `do try` and catch the thrown error